types/fluent-langneg/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for fluent-langneg 0.1
+// Project: http://projectfluent.io
+// Definitions by: Huy Nguyen <https://github.com/huy-nguyen>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export interface LanguageNegotiationOptions {
+    strategy?: 'filtering' | 'matching' | 'lookup';
+    defaultLocale?: string;
+}
+export interface NegotiateLanguages {
```

Comment:
Please use `export function negotiateLanguages(requestedLocales: ReadonlyArray<string>, availableLocales: ReadonlyArray<string>, options?: LanguageNegotiationOptions): string[];` instead of a callable interface.